fe/fe-core/src/main/java/com/starrocks/lake/LakeTableHelper.java

```diff
@@ -98,6 +99,7 @@ static boolean removeShardRootDirectory(ShardInfo shardInfo) {
         TNetworkAddress address = new TNetworkAddress(node.getHost(), node.getBrpcPort());
         try {
             LakeService lakeService = BrpcProxy.getLakeService(address);
+            MetricRepo.COUNTER_LAKE_SERVICE_RPC.increase(1L);
```

Comment:
is it possible to wrap the LakeServiceImpl and BackendServiceImpl to hook this rpc interface invoke, so the counter can be set inside the wrapper instead of adding everywhere.

Reply:
Introducing new wrap subclass of LakeService?

Reply:
It's more easy if we can just modify jprobuf-rpc